--- a/src/Application.vala	2018-06-21 01:13:01.000000000 +0200
+++ b/src/Application.vala	2018-10-18 13:12:29.755920341 +0200
@@ -52,6 +52,8 @@
                 return;

             debug ("Creating new window");
+            Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+            Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
             if (accounts.is_empty ())
                 NewAccountDialog.open ();
             else {
