diff --git a/src/Application.vala b/src/Application.vala
index 7e4ee06..d93ddda 100644
--- a/src/Application.vala
+++ b/src/Application.vala
@@ -94,6 +94,8 @@ namespace Tootle {
             }
             
             debug ("Creating new window");
+            Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+            Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
             if (accounts.is_empty ())
                 NewAccountDialog.open ();
             else {
